### DISCUSSION TITLE
Can we have authorization for certain kiali features?

### DISCUSSION THREAD

**Discussion by jmueducn** [kiali/kiali#7551] [General]
For example, Kiali Wizards might change the API Objects, can I turn off this feature or add authorization for the feature ?

**Comment by jmazzitelli** [comment]
See the "view_only_mode" setting: https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.deployment.view_only_mode

**Reply by jmueducn** [reply]
Thanks, Great help!
